chainer/training/updaters/standard_updater.py

```diff
@@ -62,13 +76,51 @@ class StandardUpdater(_updater.Updater):
             overridden, the implementation should correctly call
             :meth:`~chainer.Optimizer.new_epoch` of each optimizer.
 
+    .. note::
+
+        From v7, ``model_device`` and ``input_device`` have replaced ``device``
+        argument.
+        For backward compatibility, the ``device`` keyword argument is also
+        accepted. This argument cannot be given along with ``model_device`` or
+        ``input_device`` arguments at the same time. Specifying ``device``
+        argument is identical to giving both ``model_device`` and
+        ``input_device`` arguments with that value.
+
     """
 
     def __init__(self, iterator, optimizer, converter=convert.concat_examples,
                  device=None, loss_func=None, loss_scale=None,
-                 auto_new_epoch=True):
+                 auto_new_epoch=True, **kwargs):
+        # `device` argument is for backward compatibility.
+        # If it's given (either positional or keyword), and the device(s) are
+        # GpuDevice, StandardUpdater skips GPU-to-GPU transfer for the model.
```

Comment:
It's probably better to elaborate on what we mean by the plural devices here. You mean the device argument and the device on which the model already resides?